gomint-api/src/main/java/io/gomint/entity/Entity.java

```diff
@@ -319,6 +319,20 @@
      */
     boolean isImmobile();
 
+    /**
+     * Set this entity to a baby
+     *
+     * @param value true if this entity should be a baby, false if not
+     */
+    void setBaby( boolean value );
```

Comment:
You cannot add this to the entity class because not every entity has a baby form. think of snowballs, arrows, bats, chickens, players, etc.

Bukkit has solved this with an Ageable interface https://hub.spigotmc.org/javadocs/spigot/index.html?org/bukkit/entity/Ageable.html

Reply:
Yeah, that's right. Altough you can also just do nothing in `#setBaby(boolean)` if it is not age able and return `false` for `#isBaby()`. But for API design reasons `Ageable` would be the better approach.

Reply:
i will consider this in a complete entity refactor